test/validators.js

```diff
@@ -6458,14 +6458,20 @@ describe('Validators', () => {
           '022698763',
           '37491987654',
           '+37494567890',
+          '+37455123456',
+          '+37477123456',
+          '+37488123456',
         ],
         invalid: [
           '12345',
-          '+37411498855',
```

Comment:
This is sort of minor "breaking change", can you move the removed test-cases to the `valid` side then?

Reply:
done